prog/vm/nexus.rb

```diff
@@ -349,7 +349,7 @@ def before_run
 
   label def wait_sshable
     addr = vm.ephemeral_net4 || vm.ephemeral_net6.nth(2)
-    out = `ssh -o BatchMode=yes -o ConnectTimeout=1 -o PreferredAuthentications=none user@#{addr} 2>&1`
+    out = `ssh -o BatchMode=yes -o ConnectTimeout=1 -o PreferredAuthentications=none -o StrictHostKeyChecking=yes user@#{addr} 2>&1`
```

Comment:
I suppose, it's kinda suspicious to use a `system()` call for this rather than `net-ssh`, or a rhizome command, or somehow not relying on guests to expose a certain service on a particular port.  It may be adequate to check that port 22 is open.

But I can't deny that `wait_sshable` did help us find that issue with a kernel/OS we actually can control.

But we really need to think about ripping this out for VMs in general, and perhaps try to gain some economy by doing it for VMs we make services for.